src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -0,0 +1,367 @@
+import { ErrorHandler, Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { cloneDeep } from 'lodash';
+import { JhiAlertService } from 'ng-jhipster';
+import { fromEvent, Observable, of, Subject } from 'rxjs';
+import { debounceTime } from 'rxjs/internal/operators';
+
+import { SERVER_API_URL } from 'app/app.constants';
+import { courseOverviewTour } from 'app/guided-tour/tours/course-overview-tour';
+import { GuidedTourSettings } from 'app/guided-tour/guided-tour-settings.model';
+import { ContentType, GuidedTour, Orientation, OrientationConfiguration, TourStep } from './guided-tour.constants';
+import { AccountService } from 'app/core';
+
+export type EntityResponseType = HttpResponse<GuidedTourSettings>;
+
+@Injectable()
```

Comment:
Is it on purpose that this is not injected in root? https://angular.io/guide/dependency-injection#create-an-injectable-service-class

Reply:
@thilo-behnke I have changed that